cromshell

```diff
@@ -207,6 +210,20 @@ function usage()
   echo -e "  Update cromwell server:"
   echo -e "   update-server          Change which cromwell server jobs will be submitted to."
   echo -e ""
+  echo -e "  Get cost for a workflow"
+  echo -e "   cost [workflow-id] [[workflow-id]...]                  Get the cost for a workflow."
```

Comment:
Should we add a check for completion time?

Reply:
Yeah… I'll add in some more fancy checks that query the cromwell server for some info.